staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -463,31 +463,19 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 }
 
 func (c *Cacher) startCaching(stopChannel <-chan struct{}) {
-	// The 'usable' lock is always 'RLock'able when it is safe to use the cache.
```

Comment:
Those commends and `successfulList` seems to be outdated since rewrite of cache state was represented as lock and not via ready struct.
See https://github.com/kubernetes/kubernetes/pull/21606/files